docs/LEARNING.md

```diff
@@ -0,0 +1,33 @@
+# Ways to learn Factor
```

Comment:
The HTML page on exercism.io already has titles, so this will give us double-titles.